packages/synapse-react-client/src/lib/utils/SynapseClient.ts

```diff
@@ -3303,19 +3304,19 @@ export const getDataAccessRequestForUpdate = (
   requirementId: string,
   accessToken: string,
 ) => {
-  return doGet<RequestInterface | RenewalInterface>(
+  return doGet<Request | Renewal>(
     ACCESS_REQUIREMENT_DATA_ACCESS_REQUEST_FOR_UPDATE(requirementId),
     accessToken,
     BackendDestinationEnum.REPO_ENDPOINT,
   )
 }
 
 // http://rest-docs.synapse.org/rest/GET/accessRequirement/requirementId/dataAccessRequestForUpdate.html
-export const updateDataAccessRequest = (
-  requestObj: RequestInterface,
+export function updateDataAccessRequest(
+  requestObj: Request | Renewal,
   accessToken: string,
-) => {
-  return doPost<RequestInterface>(
+) {
+  return doPost<typeof requestObj>(
```

Comment:
We know the return type will match the request object type, so update type signature accordingly.